WordPress/Docs/Arrays/MultipleStatementAlignmentStandard.xml

```diff
@@ -0,0 +1,44 @@
+<documentation title="Multiple Statement Alignment">
+    <standard>
+    <![CDATA[
+      When declaring arrays, there should be one space on either side of a double arrow operator used to assign a value to a key.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: correct spacing between the key and value.">
+        <![CDATA[
+$args( 'cat'<em> => </em>22, 'year'<em> => </em>$current_year );
+        ]]>
+        </code>
+        <code title="Invalid: No or incorrect spacing between the key and value.">
+        <![CDATA[
+$args( 'cat'<em>=></em>22, 'year'<em>=> </em>$current_year );
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+      In the case of a block of related assignments, more space may be inserted to promote readability.
```

Comment:
If you're updating the file anyway, I think making this one a bit less like a _mild suggestion_ would be good idea.

```suggestion
      In the case of a block of related assignments, it is recommended to align the arrows to promote readability.
```